sycl/source/detail/config.cpp

```diff
@@ -45,41 +43,92 @@ static void initValue(const char *Key, const char *Value) {
 #undef CONFIG
 }
 
+void file_ignore(FILE *file, size_t n, int delim) {
+  while (n--) {
+    const int c = fgetc(file);
+    if (c == EOF)
+      break;
+    if (delim != EOF && delim == c)
+      break;
+  }
+}
+// Finds index of the newline character in char array
+int newLineIndexR(const char *string, int lowerBound, int upperBound) {
+  int currentIndex = (upperBound + lowerBound) / 2;
+  int c = string[currentIndex];
+  if (c == '\n') {
+    return currentIndex;
+  }
```

Comment:
The code above doesn't use braces for one-statement branches. Please, use consistent code style.
```suggestion
  if (c == '\n')
    return currentIndex;
```